bundles/org.openhab.binding.plex/README.md

```diff
@@ -0,0 +1,76 @@
+# Plex Binding
+
+_Give some details about what this binding is meant for - a protocol, system, specific device._
```

Comment:
This is the placeholder README and must be updated for the binding. The README is the main source of documentation for the binding and will be displayed on the add-ons page on the OH site.

Reply:
@mlobstein who updates the read me?

Reply:
The binding author is responsible for providing the readme.